migrations/migrations.sql

```diff
@@ -5101,17 +5042,63 @@ do $$
+        values (8000, 9002, 'v6.5.4', 'drydock', 'w16', 'v6.5.4', 'w16', 'w16reqproc', false, true, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
+      end if;
+
+      -- x86_64 macOS_10.12
+      if not exists (select 1 from "runtimeTemplates" where "archTypeCode" = 8000 and "osTypeCode" = 9003 and "version" = 'v6.5.4') then
+        insert into "runtimeTemplates" ("archTypeCode", "osTypeCode", "version", "drydockOrg", "drydockFamily", "drydockTag", "defaultTaskImage", "reqProcImage", "isDefault", "isAvailable", "createdAt", "updatedAt")
+        values (8000, 9003, 'v6.5.4', 'drydock', 'u16', 'v6.5.4', 'microbase', 'm10reqproc', false, true, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
+      end if;
+
+      -- x86_64 CentOS_7
+      if not exists (select 1 from "runtimeTemplates" where "archTypeCode" = 8000 and "osTypeCode" = 9004 and "version" = 'v6.5.4') then
+        insert into "runtimeTemplates" ("archTypeCode", "osTypeCode", "version", "drydockOrg", "drydockFamily", "drydockTag", "defaultTaskImage", "reqProcImage", "isDefault", "isAvailable", "createdAt", "updatedAt")
+        values (8000, 9004, 'v6.5.4', 'drydock', 'c7', 'v6.5.4', 'c7all', 'u16reqproc', false, true, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
+      end if;
+
+      -- x86_64 RHEL_7
+      if not exists (select 1 from "runtimeTemplates" where "archTypeCode" = 8000 and "osTypeCode" = 9005 and "version" = 'v6.5.4') then
+        insert into "runtimeTemplates" ("archTypeCode", "osTypeCode", "version", "drydockOrg", "drydockFamily", "drydockTag", "defaultTaskImage", "reqProcImage", "isDefault", "isAvailable", "createdAt", "updatedAt")
+        values (8000, 9005, 'v6.5.4', 'drydock', 'c7', 'v6.5.4', 'c7all', 'u16reqproc', false, true, '2018-03-28 00:00:00+00', '2018-03-28 00:00:00+00');
+      end if;
+
       -- insert additional runtimeTemplates down here
 
       --------------------------------------------------------
       -------------- Update isDefault Templates --------------
       --------------------------------------------------------
-      if exists (select 1 from "runtimeTemplates" where "version" != 'v6.4.4' and "isDefault" = true) then
-        update "runtimeTemplates" set "isDefault" = false where "version" != 'v6.4.4' and "isDefault" = true;
+      if exists (select 1 from "runtimeTemplates" where "version" != 'v6.5.4' and "isDefault" = true) then
+        update "runtimeTemplates" set "isDefault" = false where "version" != 'v6.5.4' and "isDefault" = true;
       end if;
 
-      if exists (select 1 from "runtimeTemplates" where "version" = 'v6.4.4' and "isDefault" = false) then
-        update "runtimeTemplates" set "isDefault" = true where "version" = 'v6.4.4' and "isDefault" = false;
+      if exists (select 1 from "runtimeTemplates" where "version" = 'v6.5.4' and "isDefault" = false) then
```

Comment:
These `exists` checks seem unnecessary. The update applies the same constraints.